src/Symfony/Component/Console/Style/StyleInterface.php

```diff
@@ -47,27 +47,62 @@ public function listing(array $elements);
     public function text($message);
 
     /**
+     * Formats an info result bar.
+     *
+     * @param string|array $message
+     */
+    public function info($message);
```

Comment:
Yes, in fact I've seen that some other methods miss a correspondent mention in the interface... Ok, I'll go to remove them from the interface..